posthog/temporal/common/worker.py

```diff
@@ -39,6 +40,7 @@ async def start_worker(
         workflow_runner=UnsandboxedWorkflowRunner(),
         graceful_shutdown_timeout=timedelta(minutes=5),
         interceptors=[SentryInterceptor()],
+        activity_executor=ThreadPoolExecutor(),
```

Comment:
We should set the default `max_workers` to the same as `max_concurrent_activities` (otherwise a warning will be raised). That value would be 50. Ideally, we would expose a configuration to control both.